apps/web/pages/getting-started/[[...step]].tsx

```diff
@@ -82,6 +83,13 @@ const OnboardingPage = (props: IOnboardingPageProps) => {
     <div
       className="dark:bg-brand dark:text-brand-contrast text-emphasis min-h-screen"
       data-testid="onboarding"
+      style={
+        {
+          "--cal-brand": "#111827",
+          "--cal-brand-emphasis": "#101010",
+          "--cal-brand-text": "white",
+        } as CSSProperties
+      }
```

Comment:
Forces our brand so existing css vars dont screw over primary buttons

 (the user doesnt have a brand at this point so it can be NULL)  this prevents that